### PR TITLE
Update for release 1.4

### PR DESCRIPTION
**Pull request by yunchu**
### Summary

* update for release 1.4.0
* update changelog
* update release note

### CODE REVIEW

**Review by wonjuleee — APPROVED** [review]

**Review by vinnamkim — CHANGES_REQUESTED** [review]
Changelog should be updated

**Comment by yunchu** [comment]
@wonjuleee @vinnamkim please check the PR description. it's not ready to get review yet. just opened to check coverage reporting.

**Review by chuneuny-emily — APPROVED** [review]

File: src/datumaro/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.4.0"
+__version__ = "1.4.0rc4"
```
Comment (vinnamkim) [inline src/datumaro/version.py:1]: Is it correct? rc4?

File: src/datumaro/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "1.4.0rc3"
+__version__ = "1.4.0rc4"
```
Comment (jihyeonyi) [inline src/datumaro/version.py:1]: I guess this should be "1.4.0"

**Review by jihyeonyi — CHANGES_REQUESTED** [review]

File: src/datumaro/version.py (hunk 1 above)
Comment (yunchu) [reply src/datumaro/version.py:1]: as a final RC, rc4 will be published to pypi and verified it with Geti & OTX. after then I will be published as 1.4.0.

File: src/datumaro/version.py (hunk 2 above)
Comment (yunchu) [reply src/datumaro/version.py:1]: as a final RC, rc4 will be published to pypi and verified it with Geti & OTX. after then I will be published as 1.4.0.

**Review by vinnamkim — APPROVED** [review]

**Review by jihyeonyi — APPROVED** [review]

**Pull request merged**